src/components/Books/Book/AddBook.jsx

```diff
@@ -2,18 +2,24 @@ import React from 'react';
 import { v4 as uuidv4 } from 'uuid';
 import { useDispatch } from 'react-redux';
 import { addBook } from '../../../redux/books/books';
+import { StoreApiSend } from '../../../API/api_connect';
 
 const AddBook = () => {
   const dispatch = useDispatch();
 
-  const AddBookHandler = () => {
+  const AddBookHandler = async () => {
     const newBook = {
       id: uuidv4(),
       title: document.getElementById('title').value,
       author: document.getElementById('author').value,
     };
-    // dispatch an action and pass it the newBook object (your action's payload)
-    dispatch(addBook(newBook));
+
+    if (document.getElementById('title').value !== '' && document.getElementById('author').value !== '') {
+      await StoreApiSend(newBook.title, newBook.author, newBook.id);
+      dispatch(addBook(newBook));
+    } else {
+      alert('Please fill both inputs!!!');
```

Comment:
**Optional Changes**
The previous Code Reviewer missed it, however;
- [ ] Let us restrain from using `windows.alert` . Simply put, it is difficult to test. 
Let us try to remove it before merging.